include/cpu.h

```diff
@@ -14,11 +14,9 @@ static inline void* OSPhysicalToCached(uint32_t ofs) { return (void*)(ofs + OS_B
 inline uint64_t gettick() {
     register uint32_t tbu;
     register uint32_t tbl;
-        __asm__ __volatile__ (
-            "mftbl %0\n"
-            "mftbu %1\n"
-            ::
-            "r"(tbl), "r"(tbu));
+    __asm__ __volatile__("mftbl %0\n"
+                         "mftbu %1\n" ::"r"(tbl),
+                         "r"(tbu));
```

Comment:
this got uglier, idk why clang-format puts the mftbu and the `"r"(tbl)` on the same line. Maybe `clang-format off` for this bit? Although the indentation should still be fixed

Reply:
I revert it (and added the on/off comments)

Reply:
could you dedent it too?